.github/workflows/qa-checker.yaml

```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
```

Comment:
BTW correct me if I'm wrong - you tested the workflow with `act`, correct?

Reply:
Yes